[DOC-142] YBA smart resize (#19956)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md

Using the **Edit Universe** page, you can specify the new intent for the universe. This may include a new configuration of nodes powered by a different instance type. YugabyteDB Anywhere performs these modifications through the YB-Masters powering the universe. The YB-Masters ensure that the new nodes start hosting the tablet leaders for a set of tablets in such a way that the tablet leader count remains evenly balanced across all the available nodes.

Expansion of universes created with an on-premise cloud provider and secured with third-party certificates obtained from external certification authorities follows a different workflow. For details, see [Expand the universe](../../security/enable-encryption-in-transit#expand-the-universe).

## Smart resize

> For universes that use Google Cloud Provider (GCP) or Amazon Web Services (AWS), YBA allows you to change the VM images and increase the volume size without moving the data from the old nodes to the new nodes. This is known as smart resize and is subject to the following:

Reviewer: Smart resize is also supported for Azure cloud providers unless it is increasing the volume size for ULtra SSDs

> - Instances with ephemeral disks cannot undergo smart resize to avoid potential data loss, but smart resize to ephemeral disks is supported.

Reviewer 2: ```suggestion
- To avoid potential data loss, you can't do a smart resize of instances with ephemeral disks. Smart resize _to_ ephemeral disks is supported.
```

> - Smart resize cannot be applied to instances with ephemeral disks due to a potential loss of data, but smart resize to ephemeral disks is supported.

Reviewer 2: These sound like the same thing - rephrase?

- Smart resize cannot decrease the volume size.

> - The smart resize option is not presented if anything except the values of the **Instance Type** and the size portion of the **Volume Info** fields has been changed on the **Edit Universe** page.

Reviewer 2: ```suggestion
- You can't do a smart resize if you change any options on the **Edit Universe** page other than the **Instance Type** and the size portion of the **Volume Info** field.
```

> - If you modify the value in the **Instance Type** field or in both the **Instance Type** and the size portion of the **Volume Info** field, and then click **Save**, you will be able to choose either the migration of the universe along with its data to new nodes or smart resize, as per the following illustrations:

Reviewer 2: ```suggestion
If you change the **Instance Type** or both the **Instance Type** and the **Volume Info** size and then click **Save**, YBA gives you the option to either do a full migration of the universe and its data to new nodes, or do a smart resize, as per the following illustrations:
```

  ![Full or smart resize1](/images/ee/edit-univ-1.png)
  ![Full or smart resize2](/images/ee/edit-univ-2.png)

> - If you only modify the value in the size portion of the **Volume Info** field and click **Save**, smart resize will be performed.

Reviewer 2: ```suggestion
If you change only the **Volume Info** size and click **Save**, YBA automatically performs a smart resize.
```
